fitbenchmarking/parsing/parse_sasview_data.py

```diff
@@ -39,60 +39,68 @@ def __init__(self, fname):
         self._start_x, self._end_x = self.get_start_x_and_end_x(self._data_x)
 
         self._name = entries['name']
-        self._equation = (entries['function'].split(',', 1))[0]
-
-        self._starting_values = (entries['function'].split(',', 1))[1]
-        self._starting_value_ranges = entries['parameter_ranges']
+        tmp_equation = entries['function'].split(',', 1)[0]
```

Comment:
I'm not sure all these variables need to have `tmp` prepended - as far as I can see, they are all local variables which are used to set private attributes, so just calling them `starting_values` etc would probably suffice.